### PR TITLE
update to sassc-rails

### PR DESCRIPTION
**Pull request by chnikki**
Keep getting the sass eol error when running bundle install. The sass gem reached end of life March 26, 2019. This change is switching to use sassc-rails, which uses the new C implementation of Sass as requested in issue #278. I also added rails 6.0 to the appraisal test.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,17 +6,20 @@ rvm:
...
   - gemfiles/rails_5.0.gemfile
   - gemfiles/rails_5.1.gemfile
   - gemfiles/rails_5.2.gemfile
+  - gemfiles/rails_6.0.gemfile
 
 before_install:
 - gem update --system --no-doc
-- gem install bundler
+- gem install -v 1.17.3 bundler --no-rdoc --no-ri
+- bundle _1.17.3_ install
```
Comment (javierjulio) [inline .travis.yml]: We should remove this as Travis will run this command automatically with the specific version of bundler we have installed.
```suggestion
```

File: .travis.yml [hunk 2]
```diff
@@ -6,17 +6,20 @@ rvm:
...
   - gemfiles/rails_4.2.gemfile
   - gemfiles/rails_5.0.gemfile
   - gemfiles/rails_5.1.gemfile
   - gemfiles/rails_5.2.gemfile
+  - gemfiles/rails_6.0.gemfile
 
 before_install:
 - gem update --system --no-doc
-- gem install bundler
+- gem install -v 1.17.3 bundler --no-rdoc --no-ri
```
Comment (javierjulio) [inline .travis.yml]: The build is failing but I'm not sure if its getting to this point. Could you please update the `before_install` to the following and push to see if it helps at all?

```yaml
before_install:
  - gem update --system 
  - gem install bundler:1.17.3
```

**Review by javierjulio — COMMENTED** [review]
@cparker-ajla thanks a lot for doing this! ❤️ I saw that the builds were failing but they shouldn't be. I've suggested some small changes to see if we can get them passing. Thanks!

File: .travis.yml (hunk 2 above)
Comment (javierjulio) [reply .travis.yml]: Yikes, sorry about that @cparker-ajla. Could you try this instead:
```yaml
  - gem install bundler -v "1.17.3"
```

**Comment by javierjulio** [comment]
@cparker-ajla thanks! Now we are getting a different error. I think the problem may be related to the gemspec path and gemfiles based on what I look up for that error message. Here is a sample error from one of the builds:

```
You are trying to install in deployment mode after changing
your Gemfile. Run `bundle install` elsewhere and add the
updated gemfiles/rails_5.0.gemfile.lock to version control.

The gemspecs for path gems changed
```

File: .travis.yml (hunk 2 above)
Comment (cparker-ajla) [reply .travis.yml]: Thank you for all your help getting past this error @javierjulio!

**Review by chnikki — COMMENTED** [review]
I'll try to get get these specs updated correctly.  I don't know why sass is still showing in them.

**Comment by javierjulio** [comment]
No problem! Thanks for tackling this. Let me know if you have an update and/or need help. I'm not familiar with appraisals so I'm not sure why the builds are failing. The the lock files in `/gemfiles` don't look right though so I'm thinking something went wrong with updating them perhaps?

**Pull request closed without merging**